modules/tinymce/src/plugins/advlist/test/ts/browser/ContentEditableFalseTest.ts

```diff
@@ -0,0 +1,97 @@
+import { UiFinder } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { SugarBody } from '@ephox/sugar';
+import { TinyAssertions, TinyHooks, TinySelections, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import AdvListPlugin from 'tinymce/plugins/advlist/Plugin';
+import ListsPlugin from 'tinymce/plugins/lists/Plugin';
+
+interface ListStyle {
+  readonly type: string;
+  readonly style: string;
+}
+
+interface ListParameters {
+  readonly title: string;
+  readonly content: string;
+  readonly startPath: number[];
+}
+
+describe('browser.tinymce.plugins.advlist.ContentEditableFalseTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'advlist lists',
+    advlist_bullet_styles: 'default,lower-alpha,lower-greek,lower-roman,upper-alpha,upper-roman',
+    advlist_number_styles: 'default,circle,square',
+    toolbar: 'numlist bullist',
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ AdvListPlugin, ListsPlugin ], true);
+
+  const orderedListStyles = [ 'lower-alpha', 'lower-greek', 'lower-roman', 'upper-alpha', 'upper-roman' ];
+  const numberedListStyles = [ 'circle', 'square' ];
+
+  const OlListTypes: ListStyle[] = Arr.map(orderedListStyles, (style) => ({ type: 'ol', style }));
+  const UlListTypes: ListStyle[] = Arr.map(numberedListStyles, (style) => ({ type: 'ul', style }));
+  const listTypes = [ ...OlListTypes, ...UlListTypes ];
```

Comment:
```suggestion
  const olListTypes: ListStyle[] = Arr.map(orderedListStyles, (style) => ({ type: 'ol', style }));
  const ulListTypes: ListStyle[] = Arr.map(numberedListStyles, (style) => ({ type: 'ul', style }));
  const listTypes = [ ...olListTypes, ...ulListTypes ];
```
These are using the wrong naming convention as we use camel case, not pascal case for variable names.